### PR TITLE
Controllers specs

### PR DESCRIPTION
**Pull request by Uthmanbello**
## Highlights
- Added `rspec` as a gem
- Created `requests` spec file for all controllers
- Checked responses, rendering, and placeholder texts for all actions in the controllers

### CODE REVIEW

File: spec/requests/users_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
```
Comment (codecaiine) [inline spec/requests/users_controller_spec.rb:3]: - [ ] Your test is failed. I can't provide screenshots cause my machine is crashing.    But please make sure to make fix all the errors in your test and make them pass 👍

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Uthmanbello , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good test setup ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/users_controller_spec.rb (hunk 1 above)
Comment (Uthmanbello) [reply spec/requests/users_controller_spec.rb:3]: @codecaiine ,

Thanks a lot for the review. Below, is a screenshot of the tests on my machine. I don't understand where the problem comes from but all the tests are passing here.

screenshot controllers specs

Please, I need clarification before requesting for another review.
Thanks as always.

Comment (codecaiine) [reply spec/requests/users_controller_spec.rb:3]: @Uthmanbello Good job. Maybe the errors came from my machine. Please you can submit it again another reviewer will check it 👍

**Review by DammyShittu — APPROVED** [review]
Hi @Uthmanbello,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Uthmanbello** [comment]
@DammyShittu ,

Thanks a lot

**Pull request merged**